astrowidgets/tests/test_api.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+from astropy.io import fits
+from astropy.nddata import NDData
+from astropy.table import Table
+
+from ..core import ImageWidget
+
+
+def test_load_fits():
+    image = ImageWidget()
```

Comment:
My suggestion is to make these functions into a test class' methods. That way, you can initialize `image = ImageWidget()` only once at `setup_class()` and then use the same object throughout.

Reply:
Sounds reasonable; I'm going to keep implementing as functions for the moment so we can see whether there are any tests where we want separate `ImageWidget` instances. 

For example, it looks like setting `cuts` with a string fails in a way users will find surprising if the data hasn't been set yet. In that case the `cuts` are set to `(0, 0)` (which makes sense) but that is probably not what the user wants; to test for this I think we want a new `ImageWidget` without data for that test.